### ISSUE TITLE
ionic cordova emulate still attempts to run on device

### ISSUE THREAD

**Issue by sebastian-zarzycki-es** [ionic-team/ionic-cli#4244]
**Description:**
According to Cordova documentation, `cordova emulate is an alias for cordova run --emulator. Launches the emulator instead of device`. But if it detects a viable hardware to run on (via network), it instantly switches to running on device, even despite the `emulate` command.

Explicitely providing `--emulator`, forces the emulator, but it makes `ionic cordova emulate` redundant (no different than cordova run) and misleading (`ionic cordova emulate --emulator`?). 

**Steps to Reproduce:**
```
$ ionic cordova emulate ios
```

Output:
```
[INFO] Hardware device(s) found for ios. Using --device.
(...)
> cordova build ios --device
```

**My `ionic info`:**

```
Ionic:

   Ionic CLI                     : 5.4.9
   Ionic Framework               : @ionic/angular 4.11.5
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.1.1

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.1.0, browser 6.0.0, ios 5.1.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.1.3, (and 8 other plugins)

Utility:

   cordova-res : 0.8.1 
   native-run  : 0.2.9 (update available: 0.3.0)

System:

   Android SDK Tools : 26.1.1 (/Users/rattkin/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.2
   NodeJS            : v10.16.3 (/opt/local/bin/node)
   npm               : 6.13.2
   OS                : macOS Catalina
   Xcode             : Xcode 11.2.1 Build version 11B500
```

**Comment by sebastian-zarzycki-es** [comment]
Thanks!

**Comment by dwieeb** [comment]
Thanks for the report!

**Comment by dwieeb** [comment]
This fix is in CLI 5.4.11

**Comment by capnross** [comment]
It isn't fixed in 5.4.11; it's still doing it to me on 5.4.13. Worse still, it's because I have an iMac connected and being used as an external monitor for my 2019 MBP!
